### PR TITLE
Rjt128 profile branch

### PR DESCRIPTION
**Pull request by rjt128**
Submitting profile information for Ryan Thiessen

### CODE REVIEW

File: content/authors/rjt128/_index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+superuser: false
+
+# Role/position e.g. Graduate student
+role: GRADUATE STUDENT
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Alberta
+  url: <https://www.ualberta.ca/index.html>
```
Comment (ianabc) [inline content/authors/rjt128/_index.md:18]: Can you remove the angle brackets from this line? I think it should look like
```
https://www.ualberta.ca/index.html
```

File: content/authors/rjt128/_index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+education:
+  courses:
+  - course: BSc- Mathematical Physics
+    institution: University of Saskatchewan
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: github
+  icon_pack: fab
+  link: <https://github.com/rjt128>
```
Comment (ianabc) [inline content/authors/rjt128/_index.md:37]: Can you also remove the angle brackets here?

**Comment by ianabc** [comment]
Somehow the image filter has rotated your image! That's a but on my side but might take a little while to sort out. Do you want to try uploading it again? Sometimes cropping and uploading again is enough to work around it. If you don't want to I can just merge this PR and then swing back to look at it later when things calm down. Here is the [preview](https://deploy-preview-44--mathtopowerindustry.netlify.app/)

**Comment by rjt128** [comment]
> Changes look great but the website is doing something stupid with the image filter and it's rotating your picture

Cropped the picture and reuploaded, seems to be in the right orientation.

**Comment by ianabc** [comment]
Cool, thank's for fixing that! I'll merge it now

**Pull request merged**